spring-cloud-gcp-docs/src/main/asciidoc/datastore.adoc

```diff
@@ -386,15 +386,50 @@ This feature is not supported yet.
 
 ==== Custom GQL query methods
 
-This feature is not supported yet.
+Custom GQL queries can be mapped to repository methods in one of two ways:
+
+ * `namedQueries` properties file
+ * using the `@Query` annotation
+
+===== Query methods with annotation
+
+Using the `@Query` annotation:
```

Comment:
I don't see any tests for this. Can we have a test with an actual repository?

Reply:
This is tested in the integration test.